cordova-lib/src/plugman/install.js

```diff
@@ -273,8 +273,13 @@ function runInstall(actions, platform, project_dir, plugin_dir, plugins_dir, opt
     var platformJson = PlatformJson.load(plugins_dir, platform);
 
     if (platformJson.isPluginInstalled(pluginInfo.id)) {
+        var msg = 'Plugin "' + pluginInfo.id + '" already installed on ' + platform + '.';
```

Comment:
:nit please move this line within the if statement : `if (options.is_top_level) ...`